platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -18,7 +18,7 @@ if(COMPILER STREQUAL "ARMCLANG")
       set (RTX_LIB_PATH "${CMSIS_DIR}/RTOS2/RTX/Library/ARM/RTX_V8MMN.lib")
     endif()
 elseif(COMPILER STREQUAL "GNUARM")
-    set (S_SCATTER_FILE_NAME   "${PLATFORM_DIR}/common/gcc/tfm_common_s.ld")
+    set (S_SCATTER_FILE_NAME   "${NRF_COMMON_DIR}/gcc/nordic_nrf_s.ld")
```

Comment:
@oyvindronningstad does this let as completely by pass the common linker script?

Reply:
Yes

Reply:
Thanks. So basically, you have reverted all changed to the tfm_common.? @oyvindronningstad ?

Reply:
Not all of them. There were some general improvements there, which I want to upstream.